Bea ryan byrnes patch 1

BEA-RyanByrnes opened the pull request:
Quarterly updates for FDS site, July 2020

philipashlock commented:
We got errors because the milestone update sections for "Milestone 2.4 / 2.5 / 2.6" and for "Milestone 6.1" were duplicated and used the same ID both times. At first I thought it was an accident they were duplicated and removed, but then I realized they were duplicated to be shown for the start date and end dates so I put them back and changed their IDs.

BEA-RyanByrnes commented:
Ah, perfect, thanks, Phil.  Yes, we want them in Q1 and Q2 sections, but I forgot they each had an ID assigned.


From: Philip Ashlock <notifications@github.com>
Sent: Thursday, July 23, 2020 12:08 PM
To: GSA/data-strategy <data-strategy@noreply.github.com>
Cc: Byrnes, Ryan <Ryan.Byrnes@bea.gov>; Assign <assign@noreply.github.com>
Subject: Re: [GSA/data-strategy] Bea ryan byrnes patch 1 (#221)


We got errors because the milestone update sections for "Milestone 2.4 / 2.5 / 2.6" and for "Milestone 6.1" were duplicated and used the same ID both times. At first I thought it was an accident they were duplicated and removed, but then I realized they were duplicated to be shown for the start date and end dates so I put them back and changed their IDs.

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://github.com/GSA/data-strategy/pull/221#issuecomment-663094041>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AOE65VQ4GGHJZU6LZM5RBMLR5BN4RANCNFSM4PFIDK3Q>.
